scripts/createConfig.js

```diff
@@ -112,24 +124,36 @@ networks.forEach(network => {
+    const pinnerNetworkName = network.startsWith('rsk') ? network.substr('rsk'.length) : network;
+    console.log(pinnerNetworkName);
+    const storagePinningRepoFile = `./pinner/config/${pinnerNetworkName}.json5`;
+    const storagePinningConfig = JSON5.parse(fs.readFileSync(storagePinningRepoFile));
     const storagePinningOutFile = "./out/storagePinning-" + network + "-config.json";
     storagePinningConfig.blockchain.provider = providers[network];
     storagePinningConfig.blockchain.contractAddress = storageConfig[network].storageManager;
     fs.writeFileSync(storagePinningOutFile, JSON.stringify(storagePinningConfig, null, 4));
+    fs.writeFileSync(storagePinningRepoFile, JSON5.stringify(storagePinningConfig, null, 4));
 
     // Rooms config for Pubsub Node
-    const roomsConfig = require("./templates/roomsConfig.json");
-    const roomsOutFile = "./out/rooms-" + network + ".json";
-    storageContractAddress = storageConfig[network].storageManager.toLowerCase();
-    roomsConfig.rooms = [
-      networkIds[network] + ":" + storageContractAddress + ":" + "0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1".toLowerCase(),
-      networkIds[network] + ":" + storageContractAddress + ":" + "0xFFcf8FDEE72ac11b5c542428B35EEF5769C409f0".toLowerCase(),
-      networkIds[network] + ":" + storageContractAddress + ":" + "0x22d491Bde2303f2f43325b2108D26f1eAbA1e32b".toLowerCase(),
-      networkIds[network] + ":" + storageContractAddress + ":" + "0xE11BA2b4D45Eaed5996Cd0823791E0C93114882d".toLowerCase()
-    ];
-    fs.writeFileSync(roomsOutFile, JSON.stringify(roomsConfig, null, 4));
+    // DISABLED FOR NOW:
+    // const roomsConfig = require("./templates/roomsConfig.json");
+    // const roomsOutFile = "./out/rooms-" + network + ".json";
+    // storageContractAddress = storageConfig[network].storageManager.toLowerCase();
+    // roomsConfig.rooms = [
+    //   networkIds[network] + ":" + storageContractAddress + ":" + "0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1".toLowerCase(),
+    //   networkIds[network] + ":" + storageContractAddress + ":" + "0xFFcf8FDEE72ac11b5c542428B35EEF5769C409f0".toLowerCase(),
+    //   networkIds[network] + ":" + storageContractAddress + ":" + "0x22d491Bde2303f2f43325b2108D26f1eAbA1e32b".toLowerCase(),
+    //   networkIds[network] + ":" + storageContractAddress + ":" + "0xE11BA2b4D45Eaed5996Cd0823791E0C93114882d".toLowerCase()
+    // ];
+    // fs.writeFileSync(roomsOutFile, JSON.stringify(roomsConfig, null, 4));
   }
-  // TRIGGERS TODO: Create notifier config for cache
+  
+  if (notifierConfig[network]) {
+    cacheConfig.notifier.enabled = true;
+    cacheConfig.notifier.notifierManager.contractAddress = notifierConfig[network].notifierManager;
+    cacheConfig.notifier.staking.contractAddress = notifierConfig[network].staking;
+  }
+
   fs.writeFileSync(cacheOutFile, JSON.stringify(cacheConfig, null, 4));
+  fs.writeFileSync(cacheRepoFile, JSON5.stringify(cacheConfig, null, 4));
 });
```

Comment:
Do we also have plans to generate config-docker.json based on the environment, with correct contract address for 'notiifcationManagerContract' and the endpoint for rskendpoint here?

Reply:
will be resolved in another ticket